worlds/hk/Items.py

```diff
@@ -19,18 +19,38 @@ class HKItemData(NamedTuple):
 for item, item_data in item_table.items():
     lookup_type_to_names.setdefault(item_data.type, set()).add(item)
 
-item_name_groups = {group: lookup_type_to_names[group] for group in ("Skill", "Charm", "Mask", "Vessel",
-                                                                     "Relic", "Root", "Map", "Stag", "Cocoon",
-                                                                     "Soul", "DreamWarrior", "DreamBoss")}
-
 directionals = ('', 'Left_', 'Right_')
-
-item_name_groups.update({
+item_name_groups = ({
+    "BossEssence": lookup_type_to_names["DreamWarrior"] | lookup_type_to_names["DreamBoss"],
+    "BossGeo": lookup_type_to_names["Boss_Geo"],
+    "CDash": {x + "Crystal_Heart" for x in directionals},
+    "Charms": lookup_type_to_names["Charm"],
+    "CharmNotches": lookup_type_to_names["Notch"],
+    "Claw": {x + "Mantis_Claw" for x in directionals},
+    "Cloak": {x + "Mothwing_Cloak" for x in directionals} | {"Shade_Cloak", "Split_Shade_Cloak"},
+    "LifebloodCocoons": lookup_type_to_names["Cocoon"],
     "Dreamers": {"Herrah", "Monomon", "Lurien"},
-    "Cloak": {x + 'Mothwing_Cloak' for x in directionals} | {'Shade_Cloak', 'Split_Shade_Cloak'},
-    "Claw": {x + 'Mantis_Claw' for x in directionals},
-    "CDash": {x + 'Crystal_Heart' for x in directionals},
     "Fragments": {"Queen_Fragment", "King_Fragment", "Void_Heart"},
+    "GeoChests": lookup_type_to_names["Geo"],
+    "GeoRocks": lookup_type_to_names["Rock"],
+    "GrimmkinFlames": lookup_type_to_names["Flame"],
+    "Grubs": lookup_type_to_names["Grub"],
+    "JournalEntries": lookup_type_to_names["Journal"],
+    "JunkPitChests": lookup_type_to_names["JunkPitChest"],
+    "Keys": lookup_type_to_names["Key"],
+    "LoreTablets": lookup_type_to_names["Lore"],
+    "MaskShards": lookup_type_to_names["Mask"],
+    "Maps": lookup_type_to_names["Map"],
+    "Mimics": lookup_type_to_names["Mimic"],
+    "Nail": lookup_type_to_names["CursedNail"],
+    "PalaceLore": lookup_type_to_names["PalaceLore"],
```

Comment:
Could palace totems and journal entry have this kind of handling as well or do the lookup types not have it?

Also I'm not opposed to having this group but I think that LoreTablets should include palace tablets as well.

Reply:
I take it you mean something like this?
```             
                "PalaceJournal": [
                    "Journal_Entry-Seal_of_Binding"
                ],
                "PalaceLore": [
                    "Lore_Tablet-Palace_Throne",
                    "Lore_Tablet-Palace_Workshop",
                    "Lore_Tablet-Path_of_Pain_Entrance"
                ],
                "PalaceTotem": [
                    "Soul_Totem-Palace"
                ],
```
Or should it just be a group with just Palace? I'm not sure what the best name to call it for this though

Reply:
I think this would be fine, yeah. Though I think probably PoP totems need to be in the palacetotems group too

Reply:
I've never done path of pain so good catch.